packages/klevu-ui/src/components/klevu-facet-list/klevu-facet-list.tsx

```diff
@@ -139,6 +138,10 @@ export class KlevuFacetList {
     if (!this.useApplyButton) {
       this.filters = this.manager.filters
     }
+    if (this.shouldUpdateUrlForFacets) {
+      const searchParams = this.manager.toURLParams(new URL(document.URL).searchParams)
```

Comment:
would like to understand more, why move it here, shouldnt this functionality be in klevu-facet so if someone implements only that component,  they can add to URL?

Reply:
Because the functionality of the URL generation is part of the `klevu-face-list` component. `klevu-facet` is just dump component that renders and doesn't have much features in it.